tests/contrib/django/test_django.py

```diff
@@ -1647,3 +1648,11 @@ def start_response(status, headers):
             assert root.resource == "GET ^error-500/$"
         else:
             assert root.resource == "GET tests.contrib.django.views.error_500"
+
+
+@pytest.mark.django_db
+def test_connections_patched():
+    from django.db import connections
+
+    for conn in connections.all():
```

Comment:
we do have connections here right?

e.g. we should be able to do `assert len(connections.all()) == X` or just `assert connections.all()` ?

Reply:
Ah yeh I checked that there was at least one manually (it's called `default`) but forgot to add the assert! Will add it now.